packages/strapi-connector-bookshelf/lib/build-database-schema.js

```diff
@@ -191,7 +191,7 @@ const buildColType = ({ name, attribute, table, tableExists = false, definition,
     case 'time':
       return table.time(name, 3);
     case 'datetime':
-      return table.datetime(name);
+      return table.datetime(name, { precision: 3 });
```

Comment:
Why is it needed ? that might be a breaking change for existing apps.

Reply:
By default the precision is set to milliseconds for all databases except mysql that has a precision set to seconds.
Here I force the precision to be in milliseconds as we need this precision for the locking system. I believe this change is worth it and won't really break things I think.
Do you have an idea of a better solution?